decentraland/unity-explorer
# Feat: live events map layer

## What does this PR change?

This PR adds a layer that contains all current live events, it refreshes every 5 minutes with a new request to the events endpoint

...

## How to test the changes?



1. Launch the explorer
2. Open the map
3. Verify if there are any live events being displayed (check the events website if there are any live)
4. Check that toggling on and off the live events in the filters panel they are toggled on and off

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
